types/react-native-text-input-mask/index.d.ts

```diff
@@ -91,6 +91,16 @@ export interface TextInputMaskProps extends ReactNative.ViewProps, ReactNative.T
      */
     multiline?: boolean;
 
+    /**
+     * If true, context menu is hidden. The default value is false.
+     */
+    contextMenuHidden?: boolean;
+
+    /**
+     * Callback that is called when internal `TextInput` gets ref.
+     */
+    refInput?: (ref: React.Ref<ReactNative.TextInputComponent>) => void;
+
     /**
```

Comment:
This file seems to be organized such that any non-`TextInputProps` properties go to the top (see line 17). You may want to follow that convention